### ISSUE TITLE
Convert encoding of CID

### ISSUE THREAD

**Issue by t4ccer** [mlabs-haskell/nft-marketplace-server#22]
When receiving `CID` form `nft.storage`, `CID` is encoded in `base32` (however docs do not guarantee that it will be always the case). CID encoded in NFT metadata cannot be in `base32` due to limitation of onchain metdata string length so it has to be converted (for example to `base36`).

This is not critical as we control minting, but has to be done before launch where users can mint NFTs.

**Comment by samuelWilliams99** [comment]
https://github.com/mlabs-haskell/nft-marketplace-server/blob/main/src/NftStorage.hs#L45
https://github.com/mlabs-haskell/nft-marketplace-server/blob/main/src/NftStorage.hs#L63

**Comment by samuelWilliams99** [comment]
In second line, assume `v` is base 32, convert to base 36.
Followup ticket to this will be to discover the real encoding of v, and convert to 36 accordingly. This would require the IPFS docs
